packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -236,6 +246,7 @@ class ThemeData with Diagnosticable {
     // properties (as well as brightness, primaryColorBrightness, and primarySwatch)
     // will gradually be phased out, see https://github.com/flutter/flutter/issues/91772.
     ColorScheme? colorScheme,
+    Color? colorSchemeSeed,
```

Comment:
nit: reorder to put `colorSchemeSeed` and `brightness` above `colorScheme` since it's the recommended path

Reply:
But in this case the important one is the `colorScheme` the others are just used to generate one, so I think it makes more sense to have it first and the related non-property parameters after it.